pkg/ui/workspaces/cluster-ui/src/statementDetails/diagnostics/diagnosticsView.tsx

```diff
@@ -252,6 +262,12 @@ export class DiagnosticsView extends React.Component<
             setTimeScale={onChangeTimeScale}
             className={cx("timescale-small", "margin-bottom")}
           />
+          <p className={timeScaleStylesCx("time-label", "label-margin")}>
+            Showing explain plans from{" "}
```

Comment:
is `explain plans` correct here or should it be stmt diags?

Reply:
Should be stmt diags, updated loom.